pkgs/applications/audio/tagutil/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, lib, fetchFromGitHub
+, pkg-config, cmake, libyaml
+, jansson, libvorbis, taglib
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tagutil";
+  version = "3.1";
+
+  src = fetchFromGitHub {
+    owner = "kaworu";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-oY1aGl5CKVtpOfh8Wskio/huWYMiPuxWPqxlooTutcw=";
+  };
+
+  cmakeFlags = [
+    "../src"
+  ];
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+
+  buildInputs = [
+    libvorbis
+    libyaml
+    jansson
+    taglib
+    zlib
```

Comment:
Would adding [`flac`](https://search.nixos.org/packages?channel=unstable&show=flac&from=0&size=50&sort=relevance&query=flac) enable support for this format?

Reply:
There is a problem with the code, in the future it is possible to solve.